blog/siglens-1025x-faster-than-elasticsearch.html

```diff
@@ -0,0 +1,1075 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>SigLens | SigLens 1025x faster than Elasticsearch</title>
+    <link rel="icon" href="../favicon.svg">
+
+    <!-- Font Family -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,300;0,400;0,500;0,700;1,100&display=swap"
+        rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@1,500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="../css/bootstrap.css">
+    <!--  Main CSS File -->
+    <link rel="stylesheet" href="../css/style.css">
+    <link rel="stylesheet" href="blog.css">
+
+    <!-- Font-awesome Family -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.5.1/css/all.min.css"
+        integrity="sha512-DTOQO9RWCH3ppGqcWaEA1BIZOC6xxalwEsw9c2QQeAIftl+Vegovlnee1c9QX4TctnWMn13TZye+giMm8e2LwA=="
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
+
+    <!-- Include Prism.js CSS -->
+    <link rel="stylesheet" href="./dark-theme-prismjs.css">
+
+    <link rel="canonical" href="https://www.siglens.com/blog/siglens-1025x-faster-than-elasticsearch.html">
+    <!--OG Tags-->
+    <meta property="og:url" content="https://www.siglens.com/blog/siglens-1025x-faster-than-elasticsearch.html" />
+    <meta property="og:title" content="SigLens 1025x faster than Elasticsearch" />
+    <meta property="og:description" content="SigLens - 1025x faster than Elasticsearch. Reduce your observability costs by 90%" />
+    <meta property="og:type" content="article">
+    <meta property="og:image" content="https://www.siglens.com/assets/siglens-vs-elasticsearch.png" />
+    <meta property="og:image:secure_url"
```

Comment:
use the same image that was suggested in previous comment

Reply:
done